### ISSUE TITLE
Lyrics plugin returning false positives and Polish text again

### ISSUE THREAD

**Issue by bdb484** [beetbox/beets#4704]
I just ran the lyrics plugin on a set of about 25,000 files, and it's returning _lots_ of false positives from tekstowo, and it's appending the Polish text to the beginning and end of the lyrics, as well.

As an example, running:

`beet lyrics maggie rogers honey 
`

returns the following lyrics:

> Tekst piosenki:
> ni zai wo er bian chang ge
> wo guai guai ting gang qin sheng
> shi jian fang fu ting zhi le
> zai zhe yi ke
> bao zhe ni yong bu fang shou
> 
> ...
> 
> honey honey honey
> fang zai ni de xin li
> xia ban ge shi ji wo yong fen fen miao miao hao hao ai ni
> oh honey honey honey
> duo jin ni de huai li
> wo yao wo men ni zai yi qi
> zhe yang yi zhi tian xia qu
> Dodaj interpretację do tego tekstu »
> 
> Historia edycji tekstu
> 

It looks like these issues were spotted -- and addressed -- in #4228 and #3905. Are they back, or am I using old code somehow?

**Comment by bdb484** [comment]
Sure. Here you go:

### Problem

```sh
beet -vv ls maggie rogers honey
```

Generated this output:

```
user configuration: C:\Users\bdb48\AppData\Roaming\beets\config.yaml
data directory: C:\Users\bdb48\AppData\Roaming\beets
plugin paths:
lastgenre: Loading canonicalization tree C:\Program Files\Python311\Lib\site-packages\beetsplug\lastgenre\genres-tree.yaml
Sending event: pluginload
library database: C:\Users\bdb48\AppData\Roaming\beets\library.db
library directory: C:\Dropbox (BDB)\MML2\Music
Sending event: library_opened
Maggie Rogers - Surrender - Honey
Sending event: cli_exit
```

And returned lyrics to a different song, with Polish text appended to the beginning and end:

> Tekst piosenki:
> ni zai wo er bian chang ge
> wo guai guai ting gang qin sheng
> shi jian fang fu ting zhi le
> zai zhe yi ke
> bao zhe ni yong bu fang shou
> 
> ...
> 
> honey honey honey
> fang zai ni de xin li
> xia ban ge shi ji wo yong fen fen miao miao hao hao ai ni
> oh honey honey honey
> duo jin ni de huai li
> wo yao wo men ni zai yi qi
> zhe yang yi zhi tian xia qu
> Dodaj interpretację do tego tekstu »
> 
> Historia edycji tekstu

This problem occurred over dozens of files that I've seen so far, and likely hundreds more. 

### Setup

* OS: Windows 10 Pro
* Python version: 3.11.1
* beets version: 1.6.0
* Turning off plugins made problem go away (yes/no): 



My configuration (output of `beet config`) is:

```yaml
lyrics:
    bing_lang_from: []
    auto: yes
    force: yes
    google_API_key: REDACTED
    sources:
    - google
    - musixmatch
    - genius
    - tekstowo
    bing_client_secret: REDACTED
    bing_lang_to:
    google_engine_ID: REDACTED
    genius_api_key: REDACTED
    fallback:
    local: no
directory: C:/Dropbox (BDB)/MML2/Music/
library: C:/Users/bdb48/AppData/Roaming/beets/library.db

import:
    copy: no
    write: yes

plugins: lyrics lastgenre
lastgenre:
    auto: yes
    count: 3
    force: yes
    prefer_specific: yes
    source: album
    separator: '; '
    title_case: yes
    canonical: yes
    whitelist: yes
    min_weight: 10
    fallback: 
```

**Comment by jackwilsdon** [comment]
The latest fix for the issue (#4229) hasn't made it into any releases yet. To use it, you'll need to [upgrade to the latest source version of beets](https://docs.beets.io/en/stable/faq.html#upgrade-to-the-latest-version-of-beets) (be sure to take backups!).

**Comment by bdb484** [comment]
Thanks.

My understanding of programming is basically nothing, so maybe you can help me understand something. I see that those fixes are marked as "merged," which I thought meant they had been incorporated into whatever I would install when I use pip. Is that not correct?
